vtexdocs/devportal
# Fix/public directory

#### What is the purpose of this pull request?

To move the `public` folder (currently inside `src`) to the root directory

#### What problem is this solving?

Next.js serves the **static** files inside `public`, but it must be on the root directory. Also, since the icons are not actually static (they are React components), they shouldn't be on the `public` directory.

Documented [here](https://www.notion.so/vtexhandbook/Mover-pasta-public-para-dir-raiz-do-Dev-Portal-50bc518ab52d4589985f4c51441172fa)

#### How should this be manually tested?

- Go to [this page](https://deploy-preview-12--elated-hoover-5c29bf.netlify.app/) and verify if all icons and images are shown properly
- Try visiting [this page](https://deploy-preview-12--elated-hoover-5c29bf.netlify.app/images/landing-product.png) (you should see the image from the Newsletter section of the landing page)

#### Screenshots or example usage

#### Types of changes
